ui/src/components/DataTable/index.jsx

```diff
@@ -115,14 +115,15 @@ export default class DataTable extends Component {
       noItemsMessage,
       rowsPerPage,
       paginate,
+      ...props
```

Comment:
Ah, I think we'll need to add `onHeaderClick,` here to make sure it's not passed to the `Table` component.

```jsx
onHeaderClick,
...props,
```

Reply:
hm, that's going to be an unused variable tho. The spread trick doesn't seem that elegant anymore